demos/CMakeLists.txt

```diff
@@ -142,6 +142,14 @@ find_package(OpenCV REQUIRED COMPONENTS core highgui videoio imgproc imgcodecs)
 find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 
 add_subdirectory(thirdparty/gflags)
+add_subdirectory(thirdparty/llama.cpp)
```

Comment:
We discussed offline and decided to try https://github.com/openvinotoolkit/openvino_contrib/pull/687 instead of `llama.cpp`. The steps are:
- replace llama.cpp dependency by that tokenizer branch from contrib in the submodules and cmake
- then use a short snippet in Python to convert the tokenizer in HF to OpenVINO IR using methods implemented in that branch
- replace the code of tokenization/detokenization in the sample by OV inference of tokenization/detokenization parts produced in the previous step.

@apaniukov, please comment on the functionality of the tokenizers: is it enough to be used here?
@Wovchena, could you cooperate with @apaniukov to understand how to connect and use that branch with tokenizers?